src/redux/books/booksSlice.jsx

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  bookItem: [],
+};
+const booksSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.bookItem.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      const bookId = action.payload;
```

Comment:
- [ ] @isume295 great work on your reducer , however for you to get the id from your action you are suppose to use `action.id` instead of `action.payload`, kindly consider rectifying the mistake